flink-table/flink-sql-parser/src/main/java/org/apache/flink/sql/parser/ddl/SqlCreateTable.java

```diff
@@ -21,16 +21,19 @@
 import org.apache.flink.sql.parser.ExtendedSqlNode;
 import org.apache.flink.sql.parser.error.SqlValidateException;
 
+import org.apache.calcite.sql.ExtendedSqlRowTypeNameSpec;
```

Comment:
Can't find this class in calcite-1.21.0

Reply:
This class is introduced by Flink, but we have to put it under `org.apache.calcite.sql` package, because the super class is package accessiable.

Reply:
Yes, we would move this class to the `type` package for the next Calcite version upgrade.

Reply:
@danny0405 could you create a ticket for this under FLINK-14338?

Reply:
Sure, see FLINK-15210.